lua/plugins/filetype.lua

```diff
@@ -247,6 +247,13 @@ vis.ftdetect.filetypes = {
 		ext = { "%.lua$" },
 		mime = { "text/x-lua" },
 	},
+	mail = {
+		detect = function(file, data)
```

Comment:
Please use `detect = function(_, data)` here, otherwise luacheck reports a warning because of the unused function argument, see #918.